spec/haskell/src/SEL4/Model/StateData.lhs

```diff
@@ -351,6 +351,17 @@ The function "whileM" allows writing while loops where the loop body and the con
 > whileM :: Monad m => m Bool -> m () -> m ()
 > whileM test body = whenM test $ body >> whileM test body
 
+The function "whileLoop" emulates the definition whileLoop from NonDetMonad.thy in lib.
+
+> funOfM :: State s a -> s -> a
+> funOfM f state = let (rv,state') = runState f state in rv
```

Comment:
I don't think `funOfM` is used anywhere any more (which is good). Can you please remove?